### ISSUE TITLE
0 hashrate on rtx 3070 idle

### ISSUE THREAD

**Issue by Nobellium1** [UnamSanctam/SilentCryptoMiner#738]
I just checked my panel and noticed there's a guy mining at idle mode with 0 hashrate and he has rtx3070
![image](https://user-images.githubusercontent.com/96837088/167919161-03368a24-1135-4aff-a343-a8b50a43a688.png)

**Comment by Nobellium1** [comment]
And that's not it, I've got more people with like gtx1070ti on idle having 0 hashrate too

**Comment by Nobellium1** [comment]
And rtx2060 too

**Comment by Nobellium1** [comment]
Its not only active idle, its active too
![image](https://user-images.githubusercontent.com/96837088/167920228-a8ffa15c-6d03-455a-81ff-66f4426f133a.png)

**Comment by UnamSanctam** [comment]
Yes that doesn't really matter, the Active/Active (Idle) gets set in the same location of the code, try making a configuration without "Stealth" and see if the hashrate goes up. If it does then the "Stealth" status gets overwritten somewhere below in the flow (Stealth gets checked before Max GPU/Idle GPU checks) meaning that it's actually in "Stealth".

**Comment by Nobellium1** [comment]
![image](https://user-images.githubusercontent.com/96837088/167920917-a6390d31-9c8e-4349-b1a1-dc55ff82159b.png)
Also active windows seems not to work on any of them

**Comment by UnamSanctam** [comment]
Yes because it's running as "System", "System" doesn't have any active windows and it's impossible to get other users active foreground windows in Windows due to how session separation works.

**Comment by Nobellium1** [comment]
Also do you know why's 2miners so strange when it comes to showing hashrate
![image](https://user-images.githubusercontent.com/96837088/167921545-fda7f453-b37b-4e51-9f1f-3adea1e8a4b3.png)

**Comment by Nobellium1** [comment]
Basically my hashrate goes from 232 to almost 1ghs then back to 500mhs and then to 700

**Comment by Nobellium1** [comment]
Its really impossible that it goes to 232, at least on ethermine it would never be like that

**Comment by Nobellium1** [comment]
I'm using 2miners for like a few days now, never used it before

**Comment by UnamSanctam** [comment]
Because it's how mining works, 2miners have smaller intervals of statistics. For example, 2miners might show the average over 5 minutes while Ethermine might show the average over 1 hour, and because pools calculate the hashrate as the shares submitted over some interval a smaller interval will display greater disparity between different periods of intervals.

**Comment by Nobellium1** [comment]
I see, thanks for that info

**Comment by Flex-Mini** [comment]
idk about stealth tho

**Comment by Flex-Mini** [comment]
> Basically my hashrate goes from 232 to almost 1ghs then back to 500mhs and then to 700

same

**Comment by Flex-Mini** [comment]
![image](https://user-images.githubusercontent.com/105606399/169088597-1938d349-9fb7-4835-8cc5-a7e879368cbc.png)
![image](https://user-images.githubusercontent.com/105606399/169088645-07bbc022-1164-416c-a4c3-828dc23b9fc3.png)
etc mining broken

**Comment by UnamSanctam** [comment]
You should read what I said further above, those statistics are how mining works
>Because it's due to how mining works, 2miners have smaller intervals of statistics. For example, 2miners might show the average over 5 minutes while Ethermine might show the average over 1 hour, and because pools calculate the hashrate as the shares submitted over some interval a smaller interval will display greater disparity between different periods of intervals.

**Comment by Flex-Mini** [comment]
> You should read what I said further above, those statistics are how mining works
> 
> > Because it's due to how mining works, 2miners have smaller intervals of statistics. For example, 2miners might show the average over 5 minutes while Ethermine might show the average over 1 hour, and because pools calculate the hashrate as the shares submitted over some interval a smaller interval will display greater disparity between different periods of intervals.
> 
> And yes to disable "Stealth" you leave the field empty.

![image](https://user-images.githubusercontent.com/105606399/169163854-439c2920-fc75-4aa2-a5a8-f4b4e6d573ed.png)
is this bad

**Comment by Flex-Mini** [comment]
i have my payload set to not work on VMs but forgot about Hyper-V windows prebuilt

**Comment by Flex-Mini** [comment]
> Error just means that the miner (xmrig) couldn't connect to the pool (that xmrig isn't reporting any active pool).

yeah but the guys ip locates in some washington microsoft center, GPU is from virtual machine and intel xeon cpu, username and pc name also looks bad

**Comment by Flex-Mini** [comment]
run time 1 second so they probably were debugging my shit

**Comment by UnamSanctam** [comment]
Yes I believe so as well.

**Comment by Flex-Mini** [comment]
all my miners are offline now, i have antivirus uninstaller and miner loader if it gets deleted so what is happening, i have 9 pcs now

**Comment by UnamSanctam** [comment]
Well I don't control your computers so I can't know.

**Comment by Flex-Mini** [comment]
nvm they just online online 1/4 of the day, gonna need so spread on other countries too lol

**Comment by CryptService** [comment]
🔥Buy crypt files for the miner or for your virus https://t.me/Crypt_Service 
@Cryptex777 🔥
